test/test_datasets.py

```diff
@@ -1737,5 +1723,20 @@ def inject_fake_data(self, tmpdir, config):
         return split_to_num_examples[config["train"]]
 
 
+class SvhnTestCase(datasets_utils.ImageDatasetTestCase):
+    DATASET_CLASS = datasets.SVHN
+    ADDITIONAL_CONFIGS = datasets_utils.combinations_grid(split=("train", "test", "extra"))
+
+    def inject_fake_data(self, tmpdir, config):
+        import scipy.io as sio
+
+        num_examples = 2
```

Comment:
perhaps generate 2 images for 'train' and 3 for 'test'? I think @pmeier mentioned that this increases test coverage in another PR